### PR TITLE
Feature/readmore

### PR DESCRIPTION
**Pull request by opdev1004**
This adds read more feature in About page.

And I styled the image size to be same.

If there is nothing wrong I think this is good to be merged into development branch.

I can merge this change but I am happy to get double checked.

I used Prettier for formatting.

### CODE REVIEW

File: src/jsx/components/ReadMore.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		}
+	};
+
+	return (
+		<p>
+			<span className="text" id={"text_" + props.id}>
+				{props.text}
+			</span>
+			<span className="readmore" id={"readmore_" + props.id}>
+				.....{" "}
```
Comment (caionnx) [inline src/jsx/components/ReadMore.jsx]: Change it to "...", as mostly used ellipsis

File: src/jsx/pages/About.jsx [hunk 2]
```diff
@@ -23,6 +24,8 @@ import Daryl from "../../img/about-us/daryl-diesto.png";
 import Emma from "../../img/about-us/emma-burke.png";
 import Infra from "../../img/about-us/openlaw-infrastructure.png";
 
+const _ = require("lodash");
```
Comment (caionnx) [inline src/jsx/pages/About.jsx]: Check my comment about the readme component 😃

File: src/jsx/components/ReadMore.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const ReadMore = props => {
+	const handleReadMore = e => {
+		e.preventDefault();
+		if (e) {
+			const text = document.getElementById("text_" + props.id);
+			const readmore = document.getElementById("readmore_" + props.id);
+			text.style.height = "auto";
+			readmore.style.display = "none";
+		}
+	};
```
[the remark this answers is not in the document]
Comment (opdev1004) [reply src/jsx/components/ReadMore.jsx]: Got it :)

File: src/jsx/components/ReadMore.jsx (hunk 1 above)
Comment (opdev1004) [reply src/jsx/components/ReadMore.jsx]: Got this :D

**Pull request merged**